packages/gp2-components/src/molecules/IconWithLabel.tsx

```diff
@@ -0,0 +1,25 @@
+import { pixels } from '@asap-hub/react-components';
+import { css } from '@emotion/react';
+
+const { rem } = pixels;
+
+type IconWithLabelProps = {
+  readonly icon: JSX.Element;
+};
+const styles = css({
+  display: 'flex',
+  gap: rem(9),
+  margin: `${rem(8)} 0`,
```

Comment:
I've seen some values has to be multiple of 3, is it one of the cases? For which css rules is it applied?